crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java

```diff
@@ -275,19 +275,19 @@ public Response bulk(String request) {
     @Path("/generate/{entity}/{version}/{field}")
```

Comment:
Fixed.

On Wed, May 4, 2016 at 12:49 PM, Naveen Malik <notifications@github.com>
wrote:

> In
> crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java
> <https://github.com/lightblue-platform/lightblue-rest/pull/229#discussion_r62093830>
> :
>
> > @@ -275,19 +275,19 @@ public Response bulk(String request) {
> >      @Path("/generate/{entity}/{version}/{field}")
>
> /field/path
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-rest/pull/229/files/0bc69428bb58e29e04c0745b8fdbf722b633a7bc..5d063cbd473cde20ca7402342e095fb7394f84cd#r62093830>
>